django_unicorn/components/unicorn_view.py

```diff
@@ -509,6 +509,7 @@ def get_context_data(self, **kwargs):
                     "component_id": self.component_id,
                     "component_name": self.component_name,
                     "component_key": self.component_key,
+                    "component": self,
```

Comment:
I'm relatively new to Django (in the last year) so I'm not too sure on how bad this would be to add to the context.

Reply:
I _think_ this might be fine, but I do wonder if it exposes some data that shouldn't be available in the template. What data did you need in the template? If it was just `parent` and `children`, we could explicitly add them to the context? The other approach is add `component` like you have, but deprecate `component_id`, `component_key`, `component_name` since that would be duplicative going forward.

Reply:
I thought of deprecating those three component pieces, but figured it might break anyone dependent on them. If you're going to bump the minor version, it seems like a safer change to deprecate them.

On what I need component for: I'm using it to set the 'implied' parent. Basically I found that view was always only set to the highest level component, so it wouldn't work to pass in `parent=view` past the first level. Since this unicorn info is set in the context as it goes down the tree, it was the easiest way to get the parent to each child at each level. I could have used component_id/key/name to look up the parent again, but that seems inefficient when the parent was just called before the child. 

It does expose the whole component to the template when all I really need it for is to set the parent of each child. However I didn't think of a cleaner way that wouldn't add look ups or complications.